composables/useThemeHead.ts

```diff
@@ -0,0 +1,19 @@
+/* load theme preference before page and insert them into the head. This is to
+ * avoid flicker on load. 1st check local storage, then check media preferences
+ */
+
+export function useThemeHead() {
+  useHead({
+    script: [
+      {
+        children: `
```

Comment:
I got a warning saying the `children` is deprecated and that `textContent` or `innerHtml` is the preferred alternative but haven't changed as part of this work

<img width="664" alt="Screenshot 2024-11-12 at 5 18 05 PM" src="https://github.com/user-attachments/assets/f7f5f83a-4850-43ec-ac10-1d4a6f389e8d">